samples/browser/src/main/java/org/mozilla/samples/browser/DefaultComponents.kt

```diff
@@ -120,7 +120,7 @@ open class DefaultComponents(private val applicationContext: Context) {
         )
     }
 
-    var onFindPageClicked: (() -> Unit)? = null
+    var onFindInPageClicked: (() -> Unit)? = null
```

Comment:
@csadilek Do you want to use this PR for chatting about this lambda?

Reply:
@Amejia481 let's talk through options when you're back. I don't have a quick solution that I like.

Reply:
👍

Reply:
I did it a bit differently when integrating this into the ref browser (see PR). It's still not super nice but it avoids leaking UI by attaching it to the components object.